weed/topology/store_replicate.go

```diff
@@ -43,7 +44,9 @@ func ReplicatedWrite(masterFn operation.GetMasterFn, grpcDialOption grpc.DialOpt
 	}
 
 	if s.GetVolume(volumeId) != nil {
+		start := time.Now()
 		isUnchanged, err = s.WriteVolumeNeedle(volumeId, n, true, fsync)
+		stats.VolumeServerRequestHistogram.WithLabelValues("write").Observe(time.Since(start).Seconds())
```

Comment:
change to "localWrite", "replicatedWrite"?

Reply:
ok